src/app/win32_service.cc

```diff
@@ -151,8 +152,11 @@ void I2PService::Start(
 
 void I2PService::OnStart(
     DWORD argc,
-    PSTR *argv) {
-  // TODO(unassigned): unused args
+    PSTR * argv) {
+  std::vector<std::string> args{};
+  for (DWORD i=0; i < argc; ++i)
+    args.push_back(argv[i]);
+  kovri::core::Configuration config(args);
```

Comment:
But now you've replaced unused args with an unused local :smile:

Reply:
lol, derp. Thanks for the catch, I'll add the correct usage. Guess I rushed a bit :smile: